src/main.js

```diff
@@ -133,15 +121,16 @@ let load = base.loader({
           allowedPatterns: compilePatterns(ctx.cfg.app.allowedPatterns),
           redirectLimit: ctx.cfg.app.redirectLimit,
           ensureSSL: ctx.cfg.app.ensureSSL,
+          monitor: ctx.monitor.prefix('api'),
         },
         validator: ctx.validator,
         authBaseUrl: ctx.cfg.taskcluster.authBaseUrl,
         publish: ctx.cfg.app.publishMetaData,
         baseUrl: ctx.cfg.server.publicUrl + '/v1',
         referencePrefix: 'cloud-mirror/v1/api.json',
         aws: ctx.cfg.aws,
-        component: ctx.cfg.app.statsComponent,
-        drain: ctx.influx,
+        //component: ctx.cfg.app.statsComponent,
+        monitor: ctx.monitor,
```

Comment:
I think we're going to try to have a convention of specifying the prefix 'api' for this case. I should document that in the readme.